test/e2e/node/security_context.go

```diff
@@ -214,10 +214,11 @@ var _ = SIGDescribe("Security Context", func() {
 		e2eoutput.TestContainerOutput(ctx, f, "seccomp runtime/default", pod, 0, []string{"2"}) // seccomp filtered
 	})
 
-	ginkgo.It("should support seccomp default which is unconfined [LinuxOnly]", func(ctx context.Context) {
+	// With the graduation of SeccompDefault to GA in v1.27 we now use `RuntimeDefault` instead of `Unconfined` in the default test environment.
+	ginkgo.It("should support seccomp default which is RuntimeDefault [LinuxOnly]", func(ctx context.Context) {
```

Comment:
unrelated to this PR: this whole file has `[LinuxOnly]` tests. Should it be excluded for Windows compilation?

Reply:
I think we can, let me give it a try.